docs/enhancements/ipxe-host-boot.md

```diff
@@ -0,0 +1,125 @@
+---
+title: ipxe-host-boot
+authors:
+  - "@carbonin"
+  - “@mhrivnak”
+creation-date: 2022-02-22
+last-updated: 2022-02-22
+---
+
+# iPXE Host Boot
+
+## Summary
+
+Not all user infrastructure is able to boot isos directly, and iPXE is a commonly 
+used alternative for booting machines. We should support this by making the 
+application directly serve iPXE artifacts so that systems can boot with no 
```

Comment:
Which application?
```suggestion
Assisted Installer directly serve iPXE artifacts so that systems can boot with no 
```

Reply:
Depends on the scope. If "assisted installer" means the whole stack (including the assisted service, image service, agent, controller, etc) then yeah, this is fine.

As stated later, technically the image service will host the artifacts and assisted service will host the script so I guess "Assisted Installer" makes sense here?